.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,23 @@
+name: Bug Report
+description: Open a new bug report from a discussion
+body:
+  - type: input
+    attributes:
+      label: Related Discussion
+      description: What discussion is this issue derived from
+      placeholder: "Example: #6"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Synopsis
+      description: Provide a synopsis of the issue
```

Make this description consistent with `new-feature.yml`
```suggestion
      description: Provide a brief summary of the issue
```